<launch>
  <node name="process_cv2" pkg="ros_visualise" type="process_cv2.py" output="screen" >
    <param name="topic_sub" value="/usb_cam/image_raw" />
    <param name="topic_pub" value="/gray" />
  </node>
</launch>
